_posts/2016-05-16-hoodie-at-oscon-austin-2016.md

```diff
@@ -0,0 +1,48 @@
+---
+
+title: Hoodie at OSCON in Austin
+author: Gregor
+layout: blog-post
+comments: true
+
+---
+
+The great people OSCON invited Hoodie to this year’s OSCON in Austin, to host one of the first ever OSCON Contribute sessions.  OSCON Contribute brings Open Source projects and contributors together at the OSCON Conference.
+
+## The Hackathon
+
+![OSCON Team](/blog/images/201605/oscon-contribute.jpg)
+
+The OSCON Contribute hosts [Nadja](https://twitter.com/nayafia) and [Paul](https://twitter.com/pjf) did a great job helping to prepare and promote [the Hoodie Hackathon](https://ti.to/hoodie/oscon-meet-the-hoodies), and Gregor worked until [late at night](https://twitter.com/gr2m/status/732912247308083200) to prepare [a bunch of issues for new contributors](https://github.com/hoodiehq/camp/issues).
+
+A great side effect of the event was answering the question “What is Hoodie” over and over again. It shed some light on how we can better explain what Hoodie is (and what it’s not), why it’s relevant and to whom.
+
+We settled on “Hoodie is a backend for Offline First applications” for the event – but who knows what tomorrow will bring :)
+
+## HospitalRun.io
+
+The Hoodie Hackathon run at the same time as the one from our friends HospitalRun.io, so we decided to join forces and share a room. It was great meeting [John Kleinschmidt](https://twitter.com/jkleinsc) and [Taras Mankovski](https://twitter.com/embersherpa), discuss Offline First in general and how Hoodie could become the backend of choice for HospitalRun.io in particular. We also took the opportunity and went through the code of Hoodie’s new [Admin Dashboard](https://github.com/hoodiehq/hoodie-admin), and seems like the Ember/Hoodie setup is solid.
+
+![Gregor, Taras and John at Lambert's BBQ](/blog/images/201605/oscon-hospitalrun-barbecue.jpg)
+> Gregor, Taras and John at Lambert's BBQ after the OSCON Contribute event
+
+## Talks
+
+Our favourite talk of the event was Saron’s [“What we take for granted: Examining the barriers to contributing to open source”](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/49367) about luck & privilege. We are huge fans of [her podcast](http://www.codenewbie.org/podcast) and amazing [Code Newbie community](http://www.codenewbie.org/), and it was great to finally meet her. We also met [Maria](https://twitter.com/LadyNaggaga) and [Stacy](https://twitter.com/bitchwhocodes), all three working at Microsoft, doing amazing work, what a dream team!
+
+![Microsoft’s Dream Team](/blog/images/201605/microsofts-the-best.jpg)
+> Gregor, Stacy, Maria and Saron at the CodeNewbie booth
+
+## The exhibition
+
+In the exhibition hall, fellow Offline Camp co-organiser [Teri from IBM](https://twitter.com/EventTeri) introduced us to Marvin the robot who really is full of himself thinking he could win schnick schnack schnuck against Gregor. Turns out he couldn’t :)
```

Comment:
ah yes, but the IBM team went with Marvin to Germany and people would teach him Schnick Schnack Schnuck and they all loved it :) But will fix it